browser/lib/transport/xhrtransport.js

```diff
@@ -9,7 +9,7 @@ var XHRTransport = (function() {
 	XHRTransport.isAvailable = XHRRequest.isAvailable;
 
 	XHRTransport.checkConnectivity = function(callback) {
-		Http.Request('http://internet-up.ably.io.s3-website-us-east-1.amazonaws.com/is-the-internet-up.txt', null, null, null, function(err, responseText) {
+		Http.Request(Defaults.internetUpUrlWithoutExtension + '.txt', null, null, null, function(err, responseText) {
```

Comment:
>> If not, I will need to append the current protocol to this request when in a browser environment I assume.

> I don't know what this means.

If you look in the change below in defaults.js, I no longer include `http` in the URL.  This is intentional so that the internet up check will succeed if the current connection is over HTTPS when in a browser.  However, this will therefore now fail under Node.